### PR TITLE
{bio}[GCC/9.3.0] Pysam v0.16.0.1 w/ Python 2.7.18 + 3.8.2

### PR DESCRIPTION
**Pull request by FokkeDijkstra**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by FokkeDijkstra** [comment]
I'm not really sure what the error means. I get the feeling that it suggests that Python 2 support should be removed. Fine with me, as deepTools, for which I need this does not require Python 3. 
Can someone explain what I need to do here?

**Comment by FokkeDijkstra** [comment]
When looking into it myself I find that I have to add sanity_pip_check to the config. Unfortunately the easyconfig I based this on has a comment stating that is has been explicitly disabled for Pysam. 
So blindly changing it back does not seem a good idea.

File: easybuild/easyconfigs/p/Pysam/Pysam-0.16.0-GCC-9.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+# The Francis Crick Institute
+# Modified by: Erich Birngruber
+# Gregor Mendel Institute
+# Modified by: Pavel Grochal
+# INUITS
+
+easyblock = 'PythonPackage'
+
+name = 'Pysam'
+version = '0.16.0'
```
Comment (smoors) [inline easybuild/easyconfigs/p/Pysam/Pysam-0.16.0-GCC-9.3.0.eb:16]: @FokkeDijkstra as discussed in https://github.com/easybuilders/easybuild-easyconfigs/pull/10823,
would you mind updating to 0.16.0.1?

**Comment by smoors** [comment]
Test report by @smoors
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node359.hydra.os - Linux centos linux 7.7.1908, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz (skylake_avx512), Python 2.7.5
See https://gist.github.com/0ab8d1874442371eadd3c821b75f70d9 for a full test report.

**Pull request merged**